src/thorlabs_mff_fastcs/controllers.py

```diff
@@ -58,6 +61,56 @@ def read_hardware_v(self, response: bytes) -> int:
 protocol = ThorlabsAPTProtocol()
 
 
+@dataclass
+class ResponseCache:
+    _last_update: datetime | None = None
+    _response: str | None = None
+    _update_event: asyncio.Event = asyncio.Event()
+
+    def __post_init__(self) -> None:
+        self._update_event.set()
+
+    def check_expired(self, time_step: float) -> bool:
```

Comment:
Calling this `has_expired` might make it more explicit that it returns `True` in the case `time_step` has elapsed.

Reply:
Agree